features/search_across_libraries.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Search across all libraries
+  As a researcher, I want to be able to search across all libraries
+  so that I don't need to know the specific focus of a particular library.
```

Comment:
@ekate I don't think the feature here is the "All Collections" part, I think it's the "Library search field"

```ruby
Feature: Library search field
  In order to search within specific Library collections or across all Libraries
  As a researcher
  I want to have a search field that allows me to limit by Library.
```